.github/workflows/main.yml

```diff
@@ -239,6 +239,30 @@ jobs:
     - run: cd cranelift/codegen && cargo build --features all-arch
     - run: ci/ensure_deterministic_build.sh
 
+  # Perform various builds of Cranelift under different configurations:
+  # - only x86 backend support,
+  # - only arm64 backend support,
+  # - no debug_assertions.
+  cranelift_builds:
+    name: Build cranelift
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Install Rust
+      run: rustup update stable && rustup default stable
+      shell: bash
+    - run: cargo build --manifest-path=./cranelift/Cargo.toml --bin clif-util --no-default-features --features=cranelift-codegen/arm64
```

Comment:
Oh! I just learned that I was passing `--features` wrong when I tried to do a build entirely in-tree. Very cool :)